.ci/scripts/gren

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Is this `gren` file required for this PR?

Reply:
not yet, but let's put it in place